environments/hmcts-claude-provider.json

```diff
@@ -0,0 +1,24 @@
+{
+  "account-type": "member",
+  "codeowners": ["hw469", "hw469"],
+  "environments": [
+    {
+      "name": "development",
+      "access": [
+        {
+          "sso_group_name": "modernisation-platform",
+          "level": "developer"
+        }
```

Comment:
Based on what was filled out in the form, the `github_action_reviewer` field is missing under the access config.